### PR TITLE
Update to Decidim 0.25

### PR DESCRIPTION
**Pull request by tramuntanal**
#### :tophat: What? Why?
This PR upgrades this Decidim app to Decidim v0.25.

Upgrade notes: https://github.com/decidim/decidim/releases/tag/v0.25.2

- [decidim-action_delegator](https://github.com/CodiTramuntana/decidim-module-action_delegator.git) points to the upgrade branch because it's pending to be merged.
- [decidim-verifications-members_picker](https://github.com/gencat/decidim-verifications-members_picker) points to its `0.0.2` version.
- decidim-term_customizer points to its `release/0.25-stable` branch.
-

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,21 +13,35 @@
...
+/yarn-error.log
+.yarn-integrity
+yarn-debug.log*
+
+/config/master.key
+
+/public/packs
+/public/packs-test
+/node_modules
+/yarn-error.log
```
Comment (LauraJaimeC) [inline .gitignore:45]: Can you check the duplicate folders and files?

File: .node_version [hunk 2]
```diff
@@ -0,0 +1 @@
+16.9.1
```
Comment (LauraJaimeC) [inline .node_version:1]: This file is not required, the good one is `.node-version`

File: .slug-post-clean [hunk 3]
```diff
@@ -0,0 +1,6 @@
+node_modules
+vendor/bundle/ruby/*.*.*/gems/wkhtmltopdf-binary-*/bin/wkhtmltopdf_macos_cocoa.gz
+vendor/bundle/ruby/*.*.*/gems/wkhtmltopdf-binary-*/bin/wkhtmltopdf_ubuntu_1*
+vendor/bundle/ruby/*.*.*/gems/wkhtmltopdf-binary-*/bin/wkhtmltopdf_archlinux*
+vendor/bundle/ruby/*.*.*/gems/wkhtmltopdf-binary-*/bin/wkhtmltopdf_centos*
+vendor/bundle/ruby/*.*.*/gems/wkhtmltopdf-binary-*/bin/wkhtmltopdf_debian*
```
Comment (LauraJaimeC) [inline .slug-post-clean:6]: I think this file is not needed.

File: Gemfile [hunk 4]
```diff
@@ -2,49 +2,62 @@
...
-gem "wicked_pdf"
+gem "figaro", ">= 1.1.1"
+gem "openssl"
 
-gem "savon", "~> 2.12.0"
+# if deploying to a dedicated server
+gem "daemons"
+gem "delayed_job_active_record"
+gem "whenever"
+# elsif deploying to Heroku
```
Comment (LauraJaimeC) [inline Gemfile:35]: Maybe we can remove this comment lines.

File: Procfile [hunk 5]
```diff
@@ -0,0 +1,3 @@
+web: bundle exec puma -C config/puma.rb
+worker: bundle exec sidekiq -e ${RACK_ENV:-development} -C config/sidekiq.yml
+release: bundle exec rake db:migrate
```
Comment (LauraJaimeC) [inline Procfile:3]: I think this file is not needed

File: README.md [hunk 6]
```diff
@@ -1,32 +1,49 @@
-# Decidim Suara
+# DECIDIM.CLEAN.APP
```
Comment (LauraJaimeC) [inline README.md:1]: Ups! Can you take a look and recover Suara readme?

File: config/cable.yml [hunk 7]
```diff
@@ -2,8 +2,9 @@ development:
   adapter: async
 
 test:
-  adapter: async
+  adapter: test
 
 production:
   adapter: redis
   url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  channel_prefix: decidim_clean_app_production
```
Comment (LauraJaimeC) [inline config/cable.yml:10]: 👀

File: config/sidekiq.yml [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+  - [mailers, 4]
+  - [default, 2]
+  - [newsletter, 2]
+  - [newsletters_opt_in, 2]
+  - [conference_diplomas, 2]
+  - [decidim_events, 2]
+  - [events, 2]
+  - [user_report, 2]
+  - [block_user, 2]
+  - [metrics, 1]
```
Comment (LauraJaimeC) [inline config/sidekiq.yml:12]: I think this file is not needed.

File: config/webpacker.yml [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+  public_output_path: packs-test
+
+production:
+  <<: *default
+
+  # Production depends on precompilation of packs prior to booting for performance.
+  compile: false
+
+  # Cache manifest.json for performance
+  cache_manifest: true
```
Comment (LauraJaimeC) [inline config/webpacker.yml:65]: Remember add the preproduction environment here and in `config/webpack/preprod.js`

File: docs/upgrade_apps.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+Solve conflicts and....
+
+## 7. Run db:migrate 🤞
+
+
+# For the next times
+
+1. `git pull remote-clean_app 0.2X-stable`
+2. Solve conflicts
+3. That's all!
```
Comment (LauraJaimeC) [inline docs/upgrade_apps.md:74]: 👀

File: package.json [hunk 11]
```diff
@@ -1,7 +1,29 @@
 {
-  "name": "decidim_suara_app",
```
Comment (LauraJaimeC) [inline package.json:2]: Maybe we can keep suara's name

**Review by LauraJaimeC — CHANGES_REQUESTED** [review]
Good job, only some details.

I'm going to do QA too.

**Comment by LauraJaimeC** [comment]
Some QA:

### Consultations

imagen

imagen

File: config/webpacker.yml (hunk 9 above)
Comment (tramuntanal) [reply config/webpacker.yml:65]: I think we're deploying in PRE using the `production` environment, at least for Rails. In that case this is not needed, amb I right?

File: Gemfile (hunk 4 above)
Comment (tramuntanal) [reply Gemfile:35]: I know that these comments don't apply here. But I'd prefer to keep them to keep the app as similar as possible with the template.

**Review by LauraJaimeC — APPROVED** [review]
LGTM! :)

**Pull request merged**